aggregate_spatial_binary.json

```diff
@@ -1,7 +1,7 @@
 {
-    "id": "aggregate_polygon_binary",
-    "summary": "Zonal statistics for polygons using binary aggregation",
-    "description": "Aggregates zonal statistics for one or multiple polygons over the spatial dimensions. This process passes two values to the reducer. In contrast, ``aggregate_polygon()`` passes a list of values.\n\nThe process considers all pixels for which the point at the pixel center intersects with the corresponding polygon (as defined in the Simple Features standard by the OGC).\n\nThe data cube must have been reduced to only contain two raster dimensions and a third dimension the values are aggregated for, for example the temporal dimension to get a time series. Otherwise this process fails with the `TooManyDimensions` error.\n\nThe number of total and valid pixels is returned together with the calculated values.",
+    "id": "aggregate_spatial_binary",
+    "summary": "Zonal statistics for geometries by binary aggregation",
+    "description": "Aggregates statistics for one or more geometries (e.g. zonal statistics for polygons) over the spatial dimensions. This process passes two values to the reducer. In contrast, ``aggregate_spatial()`` passes a list of values.\n\n- For **polygons**, the process considers all pixels for which the point at the pixel center intersects with the corresponding polygon (as defined in the Simple Features standard by the OGC).\n- For **points** and **lines** (line strings), the process considers all the closest pixel centers.\n\nThus, pixels may be part of multiple geometries and be part of multiple aggregations.\n\nThe data cube must have been reduced to only contain two raster dimensions and a third dimension the values are aggregated for, for example the temporal dimension to get a time series. Otherwise this process fails with the `TooManyDimensions` error.\n\nThe number of total and valid pixels is returned together with the calculated values.",
```

Comment:
How is this different from `aggregate_spatial`? And how is it possible to pass only two values altogether? Is it passing two values for every pair in the list in `aggregate_spatial`?

Reply:
This has a technical background. If lists get to large, especially for UDFs, you'd need to use binary reduction. Yes, it's basically passing the first two values in the list, then the result and the third, then the new result and the fourth and so on...

Reply:
Ah, OK. Would be nice if this could be clarified a bit in the description because it's confusing for me as a user...

Reply:
I'll check how I can do that.

Reply:
I have tried to improve the description.